main.go

```diff
@@ -234,14 +237,21 @@ func main() {
 		}
 	}
 
+	renewDeadline := time.Duration(leaderRenewSeconds) * time.Second
+	leaseDuration := time.Duration(int(1.2*float64(leaderRenewSeconds))) * time.Second
+
 	// Only attempt to grab leader lock once we have an API connection.
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                  scheme,
-		MetricsBindAddress:      metricsAddr,
-		Port:                    9443,
-		LeaderElection:          enableLeaderElection,
-		LeaderElectionID:        "storageos-api-manager-leader",
-		LeaderElectionNamespace: namespace,
+		Scheme:                     scheme,
+		MetricsBindAddress:         metricsAddr,
+		Port:                       9443,
+		LeaderElection:             enableLeaderElection,
+		LeaderElectionID:           "storageos-api-manager-leader",
+		LeaderElectionNamespace:    namespace,
+		LeaderElectionResourceLock: "leases",
```

Comment:
Same comment as before:
nit: I believe you can use the constant k8s.io/client-go/tools/leaderelection/resourcelock. LeasesResourceLock